### PR TITLE
Release v0.3 Phoenix

### PR DESCRIPTION
**Pull request by r0x0d**
## What's Changed
* Shorten the summary in specfile by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/38
* Add first tests for util, runner and logger by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/33
* Document code and update readme by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/37
* Parse minimal yaml instead of raw bash script by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/39
* Add dependabot config and update golint by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/43
* Bump actions/checkout from 2 to 3 by @dependabot in https://github.com/oamg/rhc-worker-script/pull/44
* Bump google.golang.org/grpc from 1.40.0 to 1.56.2 by @dependabot in https://github.com/oamg/rhc-worker-script/pull/48
* Bump actions/setup-go from 2 to 4 by @dependabot in https://github.com/oamg/rhc-worker-script/pull/46
* Bump thatisuday/go-cross-build from 1.0.2 to 1.1.0 by @dependabot in https://github.com/oamg/rhc-worker-script/pull/45
* Tidy up the modules and replace deprecated call WithInsecure by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/49
* Add option to create config for rhc-worker-bash by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/50
* Use LogFileName instead of LogDir for logger filename by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/52
* Fix build for go1.16 by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/53
* Revert "Bump thatisuday/go-cross-build from 1.0.2 to 1.1.0" by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/56
* Update README by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/55
* spec: improve also for EL 7 compatibility by @ptoscano in https://github.com/oamg/rhc-worker-script/pull/51
* Remove option to configure log destination by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/57
* Refactor server.go file and add tests by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/58
* Use separate environment for every executed command by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/59
* Update to make the yaml-file more generic by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/63
* Rewrite all tests with table driven approach by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/62
* Remove unnecessary logging by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/66
* Expected yaml structure should be list on top level by @andywaltlova in https://github.com/oamg/rhc-worker-script/pull/65
* Add setup for sos report by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/64
* Update the worker to make it more generic by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/67
* Rename rhc-worker.yml to rhc-worker-script.yml by @r0x0d in https://github.com/oamg/rhc-worker-script/pull/68

## New Contributors
* @dependabot made their first contribution in https://github.com/oamg/rhc-worker-script/pull/44
* @ptoscano made their first contribution in https://github.com/oamg/rhc-worker-script/pull/51

### CODE REVIEW

**Review by andywaltlova — APPROVED** [review]

File: packaging/rhc-worker-script.spec [hunk 1]
```diff
@@ -71,6 +71,17 @@ install -D -d -m 755 %{buildroot}%{rhc_worker_conf_dir}
 
 %changelog
 
+* Thu Aug 10 2023 Rodolfo Olivieri <rolivier@redhat.com> 0.3-1
+- Parse minimal yaml instead of raw bash script
+- Tidy up the modules and replace deprecated call WithInsecure
+- Add option to create config for rhc-worker-bash
+- Fix build for go1.16 by @r0x0d in #53
```
Comment (SpyTec) [inline packaging/rhc-worker-script.spec:78]: I wouldn't link like this to PRs as it's difficult to see it locally when checking a package changelog. I'd just go with the summary and remove the `by person` as well

Comment (r0x0d) [reply packaging/rhc-worker-script.spec:78]: Oh, that's true! Forgot that one.

Comment (r0x0d) [reply packaging/rhc-worker-script.spec]: done

**Review by SpyTec — APPROVED** [review]
LGTM

**Pull request merged**